.github/workflows/rosa-cluster-create.yml

```
@@ -39,12 +42,18 @@ jobs:
           aws-default-region: ${{ vars.AWS_DEFAULT_REGION }}
           rosa-token: ${{ secrets.ROSA_TOKEN }}
 
+      - name: Determine cluster name
+        run: |
+          CLUSTER_NAME_USER_INPUT=${{ inputs.clusterName }}
+          ACTUAL_CLUSTER_NAME=${CLUSTER_NAME_USER_INPUT:-"gh-${{ github.triggering_actor }}"}
```

Should be based on the owner of the fork of `keycloak-benchmark` instead of who triggered the job:

```suggestion
          ACTUAL_CLUSTER_NAME=${CLUSTER_NAME_USER_INPUT:-"gh-${{ github.github.repository_owner }}"}
```